scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
+    fs.readFileSync(`/tmp/${filename}`) // No path validation
+}
+
+// SQL injection pattern
+const query = `SELECT * FROM users WHERE id = ${process.argv[2]}` // Unsafe SQL
+
```

Comment:
:stop_sign: **Security Vulnerability**: This code is vulnerable to SQL injection attacks[^1]. Never directly interpolate user input into SQL queries.
```suggestion
import { Pool } from 'pg'

const pool = new Pool() // Assuming you've set up your PostgreSQL connection pool

async function getUserById(id: string) {
    const query = 'SELECT * FROM users WHERE id = $1'
    const values = [id]
    const result = await pool.query(query, values)
    return result.rows[0]
}

// Usage:
// const user = await getUserById(process.argv[2])
```

[^1]: CWE-89: Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection') - https://cwe.mitre.org/data/definitions/89.html